### PR TITLE
create /api/articles endpoint

### PR DESCRIPTION
**Pull request by ViktoriiaPavliukh**

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -49,7 +50,40 @@ describe('app', () => {
...
-  describe('/api/articles/:article_id', () => {
+  describe("api/articles", () => {
+    test("200: responds with an array of articles object", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+          const { articles } = response.body;
+          expect(articles).toBeInstanceOf(Array);
+          articles.forEach((article) => {
```
Comment (Augs0) [inline __tests__/api.test.js:62]: We need to check the length of arrays before ever iterating over them

File: controllers/article.controllers.js [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
+  selectArticleById,
+  selectArticles,
 } = require("../models/article.models");
 
+exports.getArticles = (req, res, next) => {
+  selectArticles().then((articles) => {
+    res.status(200).send({
+      articles: articles,
+    });
+  });
```
Comment (Augs0) [inline controllers/article.controllers.js]: We're missing a catch block here

**Review by Augs0 — CHANGES_REQUESTED** [review]
Very nice work 😄  you are missing two tiny things: thr length check in your 200 test and a catch block in your controller. Please add these then you can merge ❇️

**Review by nc-poppy-m — APPROVED** [review]
Looks great! Merge when ready

**Pull request merged**